libs/designsystem/icon/src/icon.component.ts

```diff
@@ -18,6 +17,9 @@ export enum IconSize {
   LG = 'lg',
 }
 
+const CUSTOM_NAME_DEPRECATION_WARNING =
+  'Deprecation warning: The customName input property for "kirby-icon" is getting deprecated and will be removed in v10. We therefore highly recommend you to use the name input property for custom icons instead';
```

Comment:
```suggestion
  'Deprecation warning: The customName input property for "kirby-icon" is deprecated and will be removed in v10. The name input property already supports icons registered via the iconRegistryService, and should be used instead of customName.';
```